styles.css

```diff
@@ -81,8 +88,52 @@ header {
 	text-transform: uppercase;
 }
 
+main {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-evenly;
```

Comment:
Do you specifically want space evenly? Even when expanded there is no space around the flex items, I'm not sure this line is doing anything.